server/publicapi/items/service.py

```diff
@@ -26,6 +28,60 @@ class ItemsService(BaseService):
     Serves mainly as a proxy to the data layer.
     """
 
+    def find_one(self, req, **lookup):
+        """Retrieve a specific item.
+
+        :param req: object representing the HTTP request
+        :type req: `eve.utils.ParsedRequest`
+        :param dict lookup: requested item lookup, contains its ID
+
+        :return: requested item (if found)
+        :rtype: dict or None
+        """
+        request_params = req.args or {}
+        params = list(request_params.keys())
+
+        if len(params) > 0:
```

Comment:
I agree with factoring the parameter checking out into a separate method, it's already planned and will be done when we start adding semantic checking as well (i.e. checking the values and the format of the parameters received).

Re: "query" - I don't know, ask @hlmnrmr, that's what it's called in the API specification. :smile: I would support changing it to a better (read: more explicit) name.